### PR TITLE
BOAC-3014, limit Admin on Drop-in Appt views; small fixes to /admin

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/BOAC-3014

Admins cannot:
* toggle advisor status
* check-in appointments

Admins can see appt details and cancel appts.

### CODE REVIEW

File: src/components/appointment/DropInAppointmentDropdown.vue [hunk 1]
```diff
@@ -25,41 +25,61 @@
...
+          </b-dropdown-item-button>
+          <b-dropdown-item-button
+            :id="`btn-appointment-${appointment.id}-cancel`"
+            @click="openCancelAppointmentModal(appointment)">
+            <span aria-hidden="true" class="text-nowrap">Cancel Appt</span>
+            <span class="sr-only">Cancel Appointment</span>
+          </b-dropdown-item-button>
+        </b-dropdown>
+      </div>
+    </div>
```
Comment (johncrossman) [inline src/components/appointment/DropInAppointmentDropdown.vue]: Use `?w=1` to see simple diff above.

File: boac/api/util.py [hunk 2]
```diff
@@ -139,6 +139,7 @@ def authorized_users_api_feed(users, sort_by=None, sort_descending=False):
                 'isScheduler': m.is_scheduler,
                 'automateMembership': m.automate_membership,
             })
+        profile.pop('departments')
```
Comment (paulkerschen) [inline boac/api/util.py]: @johncrossman Does this have any effect if the key gets reassigned on the next line?

Comment (johncrossman) [reply boac/api/util.py]: @paulkerschen - I did verify locally – no effect on reassignment below. Without the `pop`, the line below was not doing as I expected.

Comment (johncrossman) [reply boac/api/util.py]: @paulkerschen - I'll amend this with a better approach.

**Comment by johncrossman** [comment]
@paulkerschen - Amended and green.

**Pull request merged**